plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobContainer.java

```diff
@@ -432,6 +429,18 @@ void executeMultipartUpload(final S3BlobStore blobStore,
         }
     }
 
+    // non-static, package private for testing
+    void assertMultipartUploadSize(final long blobSize) {
```

Comment:
We shouldn't call this `assert` when it's not an `assert`. Maybe just call it `ensureMultiPartUploadSize`? :)

Reply:
Sure!